app/views/edit/build-config.html

```diff
@@ -143,74 +143,30 @@
                           </div>
 
                           <div class="form-groups" ng-show="sources.images">
-
                             <div class="single-image-source" ng-if="sourceImages.length === 1">
+
                               <div class="form-group">
-                                <label for="imageSourceFrom">Image Source From</label>
-                                <select
-                                  class="form-control simpleselector"
-                                  ng-model="imageSourceOptions.pickedType"
-                                  ng-options="type.id as type.title for type in imageSourceTypes"
-                                  ng-change="assambleInputType('imageSource', imageSourceOptions.pickedType)">
-                                </select>
+                                <label>Image Source From</label>
+                                <ui-select required ng-model="imageOptions.fromSource.type" search-enabled="false">
```

Comment:
You have `search-enabled=false` but are filtering based on `$select.search` below.